geode-core/src/distributedTest/java/org/apache/geode/distributed/internal/P2PMessagingConcurrencyDUnitTest.java

```diff
@@ -70,28 +77,71 @@
 @RunWith(GeodeParamsRunner.class)
 public class P2PMessagingConcurrencyDUnitTest {
 
-  // how many messages will each sender generate?
-  private static final int MESSAGES_PER_SENDER = 1_000;
+  // how many sending member JVMs
+  private static final int SENDERS = 1;
```

Comment:
In the course of working on this ticket, I enhanced this test so that the number of sending member/JVMs could be dialed up (larger than one). I am leaving it set at 1 for CI but wanted to leave in the ability to dial it up when we need it, for troubleshooting.

Reply:
reverted all changes to `P2PMessagingConcurrencyDUnitTest`